include/aspect/utilities.h

```diff
@@ -332,12 +332,12 @@ namespace aspect
         /**
          * Interpolation functions to access the data.
          */
-        std::vector<Functions::InterpolatedUniformGridData<dim> *> data;
+        std::vector<Functions::InterpolatedTensorProductGridData<dim> *> data;
```

Comment:
(Note: `Function<dim>` is a base class for both the uniform and tensor product classes.)